util/src/main/java/com/google/cloud/hadoop/util/CredentialConfiguration.java

```diff
@@ -30,6 +30,10 @@
   private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
 
   private boolean serviceAccountEnabled = true;
+  private String saPrivateKeyId = null;
```

Comment:
sa -> serviceAccount for consistency?

Reply:
This was really grouping these 3 related parameters for this configuration mechanism. I could rename it to serviceAccount instead of sa if you feel strongly about it.